src/index.js

```diff
@@ -447,8 +411,11 @@ function findValues(current, keypath, index, object, values) {
 /**
  */
 
-function createNestedValidator(keypath, a, q) {
-  return { a: { k: keypath, nv: a, q: q }, v: nestedValidator };
+function createNestedValidator(keypath, validator, query) {
+  const arg = {path: keypath, validator, query};
```

Comment:
Just checking - is this the only place `nestedValidator` is called and so the only place we need to change the values of `arg`?

Reply:
Yes!